stable/anchore-engine/templates/engine_configmap.yaml

```diff
@@ -256,4 +257,11 @@ data:
         ssl_cert: "/home/anchore/certs/{{- .Values.anchoreGlobal.internalServicesSsl.certSecretCertName }}"
         ssl_key: "/home/anchore/certs/{{- .Values.anchoreGlobal.internalServicesSsl.certSecretKeyName }}"
         {{- end }}
-
```

Comment:
Is this a config that should only exist for enterprise?

Reply:
@Btodhunter yeah